src/datawave_cli/authorizations_interactions.py

```diff
@@ -28,24 +29,51 @@ def authorization_whoami(self):
         """
         self.log.info("Getting the authorization details for my cert from DW...")
         request = f"{self.base_url}/authorization/v1/whoami"
-        resp = requests.get(request, cert=self.cert, headers=self.headers, verify=False)
-        log_http_response(resp, self.log)
         try:
+            resp = requests.get(request, cert=self.cert, headers=self.headers, verify=False)
+            resp.raise_for_status()
+            log_http_response(resp, self.log)
             self.log.info(json.dumps(resp.json(), indent=1))
             return resp.json()
-        except requests.decoder.JSONDecodeError:
-            self.log.info(resp.text)
+        except (HTTPError, JSONDecodeError, Timeout) as e:
+            msg = "A bad response from the endpoint whoami was found"
+            self.log.error(f"{msg}: {e}")
+            raise RuntimeError(msg) from e
+
+    def authorization_evict_users(self):
+        """
+        Calls the /admin/evictUsers within the Datawave authorization to evict
+        all users within the cache.
+        """
+        self.log.info("Requesting all users to be evicted from DW...")
+        request = f"{self.base_url}/authorization/v1/admin/evictAll"
+        try:
+            resp = requests.get(request, cert=self.cert, headers=self.headers, verify=False)
+            resp.raise_for_status()
+            log_http_response(resp, self.log)
+            return resp
+        except (HTTPError, JSONDecodeError, Timeout) as e:
+            msg = "An error occurred while requesting to evict all users"
+            self.log.error(f"{msg}: {e}")
+            raise RuntimeError(msg) from e
 
 
 def main(args):
     log = setup_logger("authorizations_interactions", log_level=args.log_level)
     ai = AuthorizationInteractions(args, log)
 
-    return ai.authorization_whoami()
+    if args.whoami:
+        return ai.authorization_whoami()
+    elif args.evict_users:
+        return ai.authorization_evict_users()
+    else:
+        log.warning("No cmd given, exiting without running anything. Pass '--whoami' or '--evict_users'.")
 
 
 @click.command
 @common_options
+@click.option("--whoami", is_flag=True, help="Flag to denote to run the whoami endpoint.")
```

Comment:
Should we make one of these the default option? Probably whoami cause that is going to be called a lot more often than evict